nextmv/tests/test_options.py

```diff
@@ -24,7 +24,7 @@ class TestOptions(unittest.TestCase):
     assume that the script is one level up.
     """
 
-    test_scripts = [1, 2, 3, 4, 5, 6, 7]
+    test_scripts = [1, 2, 3, 4, 5, 6, 7, "_deprecated"]
```

Comment:
```suggestion
    test_scripts = ["1", "2", "3", "4", "5", "6", "7", "_deprecated"]
```

Can we convert it to just strings? I know it works, is only in a test and doesn't really matter, but the mixed types in the list kind of send shivers down my spine. :joy: 

But yeah, feel free to ignore. Mainly noticing tbf. :blush:

Reply:
Yep, sorry about that.